styles/planetcaravan/layout.s2

```diff
@@ -41,6 +41,7 @@ set margins_unit = "%";
 
 propgroup colors_child {
     property Color color_entry_footer_background { des = "Entry footer background color"; }
+    property Color ccolor_entry_footer_text { des = "Entry footer text color"; }
```

Comment:
Okay, typo fixed and squashed into the previous commit

On Thu, Jun 25, 2015 at 7:18 PM, Afuna <notifications@github.com> wrote:

> In styles/planetcaravan/layout.s2
> <https://github.com/dreamwidth/dw-free/pull/1442#discussion_r33315515>:
>
> > @@ -41,6 +41,7 @@ set margins_unit = "%";
> >
> >  propgroup colors_child {
> >      property Color color_entry_footer_background { des = "Entry footer background color"; }
> > +    property Color ccolor_entry_footer_text { des = "Entry footer text color"; }
>
> psst ccolor_entry
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dreamwidth/dw-free/pull/1442/files#r33315515>.
>